docs/general/session.md

```diff
@@ -35,4 +35,41 @@ backends can link the two sessions.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Session Events
```

Comment:
From an overall semconv maintainer standpoint - I *would* like to avoid the state where we can add conventions without YAML / machine-readable backing.

Is the event and/or client WG tracking all of these contributions with plans to immediately move to YAML when able?

If, not could you at least define an empty `event` group with the id of this event?

I'd also be happy if you updated `Event` template to declare that `event.name` gets filled out with the name of the group in the YAML: https://github.com/open-telemetry/semantic-conventions/blob/main/templates/registry/markdown/event_macros.j2#L2

Reply:
I've created https://github.com/open-telemetry/semantic-conventions/issues/1133 to track these events that are being described in markdown before the yaml tooling is ready.